roles/openshift_openstack/tasks/generate-templates.yml

```diff
@@ -10,6 +10,10 @@
     stack_template_path: "{{ stack_template_pre.path }}/stack.yaml"
     user_data_template_path: "{{ stack_template_pre.path }}/user-data"
 
+- name: set stack name
+  set_fact:
+    openshift_openstack_stack_name: "{{ lookup('env', 'OPENSHIFT_CLUSTER') | ternary (lookup('env', 'OPENSHIFT_CLUSTER'), omit) }}"
```

Comment:
Set fact is kind of hard to track and we don't generally allow env lookup in roles as that can be confusing to the role users.

Please set the default in `roles/openshift_openstack/defaults/main.yml` to the same value as in the inventory (e.g. `openshift-cluster`) and put the env lookup to the `import_role` call in `playbooks/openstack/openshift-cluster/provision.yml`.

That way the standalone users of this role will have full control over the value from Ansible.

Reply:
Done